docs/upgrades.md

```diff
@@ -57,12 +57,24 @@ assumption of the upgrade script will break.
 
 ### Changing the Version
 
-The version file is specified in `detect_secrets.__version__`. Just modify this value to the
-desired version, based on your change that is made.
+`detect-secrets` integrates with `bump2version` to make this process easier. Here's a handy
```

Comment:
I'd rather this work like all the other Yelp packages (merge to master, bump version number, tag the new version, let CI do the release)

If you want to write a convenience script for doing version bumping/changelog generation and tagging go ahead, but the actual release should be CI

Reply:
I'd love to get a working CI again (after travis-ci.com stopped working for us), but that's going to have to wait for when we have the bandwidth to do infra improvements. Until then, scrappy solutions will need to suffice.